src/server/test_utils.cc

```diff
@@ -209,13 +209,16 @@ RespExpr BaseFamilyTest::Run(absl::Span<std::string> span) {
 RespExpr BaseFamilyTest::Run(std::string_view id, ArgSlice slice) {
   TestConnWrapper* conn_wrapper = AddFindConn(Protocol::REDIS, id);
 
+  conn_wrapper->conn()->SetAdmin(admin_);
```

Comment:
Could we set it directly in `BaseFamilyTest::SetAdmin()` instead?
(to clarify: drop `BaseFamilyTest`'s member `is_admin_` and only use the test connection's one

Reply:
I can set the Admin connection from RunAdmin , it will need to duplicate some code from the Run logic, but I think its ok.
Pushing a fix